packages/flutter/lib/src/material/slider.dart

```diff
@@ -925,6 +927,7 @@ class _RenderSlider extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     required TextDirection textDirection,
     required bool hasFocus,
     required bool hovering,
+    required DeviceGestureSettings gestureSettings,
```

Comment:
Should not `this.gestureSettings`?

Reply:
Since the only thing I do with `gestureSettings` is pass it to the gesture detectors below, we don't necessarily need a field to store the settings on this render object. The gesture detectors themselves are `late` and so aren't re-created outside of the constructor

Reply:
oh, you are right, : )